content/applications/productivity/discuss.rst

```diff
@@ -1,105 +1,173 @@
 :show-content:
-:show-toc:
+:hide-page-toc:
 
 =======
 Discuss
 =======
 
-Odoo *Discuss* is an internal communication app that allows users to connect through messages,
-notes, and file sharing, either through a persistent chat window that works across applications, or
-through the dedicated *Discuss* dashboard.
+Odoo **Discuss** is an internal communication app that allows users to connect through messages,
+notes, file sharing, and video calls. **Discuss** enables communication through a persistent chat
+window that works across applications, or through the dedicated **Discuss** dashboard.
```

Comment:
Italicize "*Discuss* dashboard" instead of bold/guilabel, since the text is not on the active UI.

Also, suggestion to move the initial navigation instruction here (since the proceeding sections rely on the user being on the dashboard
```suggestion
window that works across applications, or through the dedicated *Discuss* dashboard.

Upon opening the :menuselection:`Discuss app`, the :guilabel:`Discuss` dashboard appears.
```